contents/blog/best-open-source-analytics-tools.md

```diff
@@ -18,39 +18,40 @@ tags:
   - Comparisons
 ---
 
-It's a truth universally acknowledged that every business reliant on an outmoded enterprise product has need of an open source alternative... probably.
+There's no shortage of high-quality open-source analytics tools. In this guide we've split them into two categories:
 
-In this guide we're looking at open-source analytics tools, which we've split into two broad categories:
+- [Analytics tools](#the-best-open-source-analytics-tools) that collect data and help you understand user behavior. This includes product analytics platforms, but also web analytics, session recording, and AB testing tools.
 
-- [Open-source analytics tools](#the-best-open-source-analytics-tools) that collect data and help you understand user behavior. This includes product analytics platforms, but also web analytics, session recording, and AB testing tools.
+- [Dashboarding tools](#the-best-open-source-dashboard-and-bi-tools) that connect to data sources (e.g. a data warehouse) for building dashboards, insights, interactive visualizations, and real-time data logging.
```

Comment:
Should we write an article for this instead? Both open source visualization tools and open source dashboard tools get solid (~400/m) search traffic.